main.py

```
@@ -148,213 +91,234 @@ async def send_message(self, connection_id: str, message: str):
+
+        for chunk in response:
+            await manager.send_message(connection_id, json.dumps(chunk.model_dump()))
+
+        await manager.send_message(connection_id, "[DONE]")
+    except WebSocketDisconnect:
+        logger.info(f"WS disconnected: {connection_id}")
     except Exception as e:
-        logger.error(f"WebSocket auth error (v2): {e}", exc_info=True)
-        try: await websocket.send_json({"error": "Authentication failed due to an internal error."}); await websocket.close()
-        except: await websocket.close() # Ensure close on error
+        logger.error(f"WS Error: {str(e)}")
```

Ruff (0.15.14)

[warning] Use explicit conversion flag

Replace with conversion flag

(RUF010)